src/Halogen/Components/Select/Rotary/Component.purs

```diff
@@ -0,0 +1,348 @@
+module Halogen.Components.Select.Rotary.Component
```

Comment:
I don't know what best to suggest for these modules, `purescript-halogen-menu` uses `Halogen.Menu..` so maybe we can just go with `Halogen.Select..` here too.

If we do want to keep it more like it is I'd suggest going with `Halogen.Component` rather than `Components` though. I know that namespace prefix is already used within Halgoen, but I can't think of any other cases where we pluralise a namespace.